src/lib/stores/linearTeams.ts

```diff
@@ -0,0 +1,72 @@
+import { browser } from '$app/environment';
+import { writable } from 'svelte/store';
+import type { Team } from '../linear';
+import { toast } from '@significa/svelte-ui';
+
+const getDefaultTeams = () => {
+  if (browser) {
+    const localStoreTeams = window.localStorage.getItem('linear-teams');
+
+    if (localStoreTeams) {
+      try {
+        JSON.parse(localStoreTeams);
+      } catch (e) {
+        throw new Error();
+      }
+
+      const parsedTeams = JSON.parse(localStoreTeams);
```

Comment:
Also, you're not checking if the parsed object is the actual object you are expecting it to parse.
You can't trust the data in your localStorage